lib/cfasttext.cc

```diff
@@ -133,6 +133,13 @@ void cft_fasttext_quantize(fasttext_t* handle, fasttext_args_t* args) {
     ((FastText*)handle)->quantize(*a);
 }
 
+void cft_get_word_vector(fasttext_t* handle, const char *word, float *buf) {
```

Comment:
It's unclear how large the `float *buf` should be, is it better to just return the pointer? We might also want to provide a `cft_word_vector_free(float *buf)` function?

Reply:
The receiving buffer should have enough space to contain `cft_fasttext_get_dimension()` of `float`.
This dimension is a fixed value for a fasttext model so I don't think we need to specifically set it in parameter.

Reply:
OK, do you mind adding some comments to clarify the expected usage here? It would be great to have an example.

```c
float* buf = (float*)malloc(cft_fasttext_get_dimension(handle) * sizeof(float));
cft_get_word_vector(handle, "word", buf);
// do something with the word vector
free(buf);
```